All_years/04-Social_Care/04-SDS_data.R

```diff
@@ -58,14 +58,24 @@ sds_full_clean <- sds_full_data %>%
   mutate(across(starts_with("sds_option_"), ~ .x == "1")) %>%
   # SDS option 4 is derived when a person receives more than one option.
   # e.g. if a person has options 1 and 2 then option 4 will be derived
-  mutate(sds_option_4 = rowSums(across(starts_with("sds_option_"))) > 1, .after = sds_option_3) %>%
+  mutate(
+    sds_option_4 = rowSums(across(starts_with("sds_option_"))) > 1, .after = sds_option_3,
+    # Fix sds option 4 cases where all 3 options are missing
+    sds_option_4 = if_else(sds_option_1 == FALSE & sds_option_2 == FALSE & sds_option_3 == FALSE, TRUE, sds_option_4)
```

Comment:
Simplification.
```suggestion
    sds_option_4 = if_else(!any(sds_option_1, sds_option_2, sds_option_3), TRUE, sds_option_4)
```

Reply:
I think we need to apply this to only Angus, i.e.
```suggestion
    sds_option_4 = if_else(!any(sds_option_1, sds_option_2, sds_option_3) & sending_location == "Angus", TRUE, sds_option_4)
```

Can you also double check that there are no `NA`s? I think this will be dealt with on line 58 but I'm not sure.

Reply:
In my latest commit, i have tried to deal with the NA's but i don't think this is working right as i tried to do this in a clever way. Any suggestions to improve this? code below

 `mutate(across(starts_with("sds_option_"), ~ if_else(is.na(.x), FALSE,.x)))`